### ISSUE TITLE
Incorrect handling of decimally specified terminals (%dXX)

### ISSUE THREAD

**Issue by gandaro** [michaelrsweet/abnf#12]
## Expected behavior
`%d42` should yield the character `*`, ASCII character 42

## Actual behavior
`%d42` yields `B`, ascii character 66 (0x42)

## Cause
In [lines 484-489](https://github.com/michaelrsweet/abnf/blob/68219f89dbe235e8cde062246078ce4683f36832/abnf-functions.php#L484-L489) of abnf-functions.php, it says
```php
	 /* Decimal */
	  while ($i < $len && ctype_digit($token[$i]))
	  {
	    $ch = $ch * 16 + (int)$token[$i];
	    $i ++;
	  }
```
Note that `$ch` is multiplied by the wrong base, namely 16 instead of 10.

**Comment by michaelrsweet** [comment]
Whoops, wonder how I missed this one for so long?!?

**Comment by michaelrsweet** [comment]
[master 1ed90c5] Fix decimal terminals (Issue #12)
